src/main/java/pianokeys/aws/CompositionRequestHandler.java

```
@@ -15,23 +20,35 @@ public class CompositionRequestHandler implements RequestHandler<APIGatewayProxy
 
     private final Gson gson = new Gson();
 
+    private final Playlist playlist = new Playlist(); // this is where the playlist will be saved to
+
     @Override
     public APIGatewayProxyResponseEvent handleRequest(APIGatewayProxyRequestEvent event, Context context)
     {
         try
         {
-            // Retrieve the body and change json into an object
+            String method = event.getHttpMethod();
             String body = event.getBody();
-            // CompositionRequest request = gson.fromJson(body, CompositionRequest.class);
-
-            // Do something with the request and create a CompositionResponse
-            // CompositionResponse response = new CompositionResponse("Received composition request");
 
-            // Create the HTTP response with the CompositionResponse
-            // String responseJson = gson.toJson(response);
             APIGatewayProxyResponseEvent apiResponse = new APIGatewayProxyResponseEvent();
-            apiResponse.setStatusCode(200);
-            // apiResponse.setBody(responseJson);
+
+            if ("POST".equals(method))
+            {
+                return handlePost(body);
+
+            } else if ("PUT".equals(method))
+            {
+                return handlePut(body);
+
+            } else if ("DELETE".equals(method))
+            {
+                return handleDelete(body);
+
+            } else if ("GET".equals(method))
+            {
+                return handleGet();
+            }
+
             return apiResponse;
```

Throw an Exception instead of returning a blank apiResponse